Make default value for $XFILESEARCHPATH available to other components,
such as xlock, without having to link with libXt.

--- libXt-1.1.5/xt.pc.in
+++ libXt-1.1.5/xt.pc.in
@@ -5,6 +5,7 @@ includedir=@includedir@
 appdefaultdir=@appdefaultdir@
 datarootdir=@datarootdir@
 errordbdir=@ERRORDBDIR@
+xfilesearchpath=@XFILESEARCHPATHDEFAULT@
 
 Name: Xt
 Description: X Toolkit Library
